pgo/cmd/auth.go

```diff
@@ -175,15 +178,31 @@ func GetCredentialsFromEnvironment() msgs.BasicAuthCredentials {
 	}
 	return creds
 }
-func GetCredentials() {
-	log.Debug("GetCredentials called")
+
+// GetSessionCredentials gathers the pgouser and password information
+// and stores them for use by the PGO client
+func GetSessionCredentials(){
+	log.Debug("GetSessionCredentials called")
 
 	SessionCredentials = GetCredentialsFromEnvironment()
 
 	if !SessionCredentials.HasUsernameAndPassword() {
 		SessionCredentials = GetCredentialsFromFile()
 	}
+}
+
+// GetCredentials instantiate gathers the necessary client connection 
+// TLS settings for use by the PGO client.
+// If TLS is disabled, this will not be called.
+func GetCredentials() *http.Transport{
+	log.Debug("GetCredentials called")
 
+	SessionCredentials = GetCredentialsFromEnvironment()
+
+	if !SessionCredentials.HasUsernameAndPassword() {
+		SessionCredentials = GetCredentialsFromFile()
+	}
+		
```

Comment:
I don't believe this is needed here?

Reply:
Correct, this is called earlier in the new function. This instance is now removed.